model-optimizer/mo/front/tf/loader.py

```diff
@@ -215,7 +215,7 @@ def load_tf_graph_def(graph_file_name: str = "", is_binary: bool = True, checkpo
         if model_dir:
             # saved model directory
             try:
-                env_setup = get_environment_setup()
+                env_setup = get_environment_setup("tf")
```

Comment:
You have to update other loaders for Caffe, ONNX, MxNet etc.

Reply:
The function use only in tf loader. For other loaders get_environment_setup do not call.